src/telemetry/telemetry.controller.ts

```diff
@@ -11,13 +11,41 @@ import {
 } from '@nestjs/common';
 import { ApiExcludeEndpoint } from '@nestjs/swagger';
 import { Request } from 'express';
-import { gte, valid } from 'semver';
+import semver from 'semver';
 import { ApiConfigService } from '../api-config/api-config.service';
 import { InfluxDbService } from '../influxdb/influxdb.service';
+import { CreatePointOptions } from '../influxdb/interfaces/create-point-options';
 import { NodeUptimesService } from '../node-uptimes/node-uptimes.service';
 import { UsersService } from '../users/users.service';
 import { VersionsService } from '../versions/versions.service';
-import { WriteTelemetryPointsDto } from './dto/write-telemetry-points.dto';
+import {
+  WriteTelemetryPointDto,
+  WriteTelemetryPointsDto,
+} from './dto/write-telemetry-points.dto';
+
+/** How many blocks per sequence for block_propagation measurement to allow through telemetry */
+export const BLOCK_PROPAGATION_INTERVAL = 5;
+
+const TELEMETRY_WHITELIST = new Map<string, true | RegExp | Array<string>>([
```

Comment:
nit: I don't feel strongly about this, but if we're not using any `RegExp` now maybe we can remove it and add it in the future.

Reply:
I think it'll be needed by someone soon, but I generally agree with the premise of no code that is not planned to be used very soon. I'll remove it for now.

Reply:
Fixed